<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTDialog</class>
 <widget class="QDialog" name="DTDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结构元素类型选择：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="squareLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>122</width>
     <height>123</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="crossLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>122</width>
     <height>123</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结构元尺寸：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5x5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7x7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9x9</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="diskButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>圆盘形</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="crossButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>十字架形</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="squareButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>57</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>方形</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="diskLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>122</width>
     <height>123</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
